src/test/deadlock_tests/suite.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2019 Greg Griffith
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "sync.h"
+
+#include "test/test_bitcoin.h"
+
+struct EmptySuite
+{
+    EmptySuite()
+    {
+        ECC_Start();
+        SetupEnvironment();
+        SetupNetworking();
```

Comment:
It doesn't have anything i "need" to avoid. I actually don't need anything in the suite setup just an empty suite to fill the requirement that one be passed into boost. This empty suite should be empty but i probably messed up a rebase somewhere along the way. 

I can either remove the rest of the functions in this suite or remove this suite entirely and just use the BasicTestingSetup depending on everyones agreed preference.

Reply:
i removed the functions so it is now properly empty

Reply:
Did you test that EmptySuite works in isolation? Like for example ./test_bitcoin -t test8

I’ve experienced that not doing all that boilerplate initializing stuff leads to mysterious crashes

Reply:
yes. i run the entire ./test_bitcoin as well as just the deadlock tests (using --run_test) whenever i make changes. nothing has given me odd crashes that were related to the empty suite